src/data/linux-quiz.ts

```diff
@@ -1,4 +1,25 @@
 const linuxQuiz = [
+  {
+    Question: "What does the -q option do in most command-line tools?",
+    Answer: "Suppresses output",
+    Distractor1: "Enables verbose output",
+    Distractor2: "Runs the command in the background",
+    Distractor3: "Stops the command execution",
+    Explanation:
+      "In most command-line tools, the -q or --quiet option is used to suppress the output of the command, making the execution silent unless there are errors.",
+    Link: "https://man7.org/linux/man-pages/man1/rsync.1.html"
+  },
+
+  {
+    Question: "Which of the following is not a mode in the Vim editor?",
+    Answer: "Write",
+    Distractor1: "Insert",
+    Distractor2: "Visual",
+    Distractor3: "Command",
+    Explanation:
+      "In the Vim editor, there is no mode called 'Write.' The modes include Insert, Visual, and Command. 'Write' refers to a command (':w') used to save changes, not a mode.",
+    Link: "https://www.geeksforgeeks.org/entering-editing-mode-in-vim-editor/"
```

Comment:
We don't accept links from external sources like geeks for geeks.
We only accept links from freeCodeCamp news publication, freeCodeCamp youtube videos, MDN or official docs.
Please change this

Reply:
Thank you for pointing that out , I fixed it!